Cargo.toml

```diff
@@ -2,5 +2,12 @@
 members = ["relay", "relay-*", "tools/*"]
 default-members = ["relay"]
 
+[profile.dev]
```

Comment:
This seems like counterintuitive behavior. Do we want to start doing this across the org?

Reply:
I think we should do this only where it shows significant benefits. I can imagine doing this in symbolicator, although there stack traces are actually useful for debugging in contrast to Relay. Repositories like `symbolic` or even smaller crates build fast enough so that such a hack is not necessary.

For the curious: This cuts build times in half, and reduces the size of the target folder by 45%.